app/validators/digital_object/dynamic_fields_validator.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+# This class provided the logic for validating dynamic fields against a map generated by DynamicFieldMap.
+class DigitalObject::DynamicFieldsValidator < ActiveModel::EachValidator
+  CORE_TERM_FIELDS = ['uri', 'alt_labels', 'pref_label', 'term_type', 'authority', 'term_type'].freeze
```

Comment:
How would you feel about putting this constant in the Term class?  It's not wrong to have it here, but it feels to me like something that should be defined in that context -- sort of "owned by" the Term class.  This is just a slight preference, so I'm find keeping it here if you'd prefer having it here.

Reply:
I can move it out!

Reply:
Thanks!